src/main/java/tds/exam/results/mappers/OpportunityMapper.java

```diff
@@ -49,6 +49,7 @@
             .collect(Collectors.toList());
 
         TDSReport.Opportunity opportunity = new TDSReport.Opportunity();
+        opportunity.setOppId("0");
```

Comment:
Can you also add a comment why we need to set the OppId to 0 rather than the exam id UUID?